docs/developers-guide/driver-changelog.md

```diff
@@ -4,6 +4,14 @@ title: Driver interface changelog
 
 # Driver Interface Changelog
 
+## Metabase 0.50.0
+
+- The multimethod `metabase.driver/describe-fields` has been added. The method needs to be
+  implemented if the database supports the `:fast-sync-fields` feature.
+
+- The multimethod `metabase.driver/describe-fks` has been added. The method needs to be implemented if the database
+  supports the `:foreign-keys` and `:fast-sync-fks` features.
+
```

Comment:
Do you mean for this PR to be stacked on #38970? It seems to include it.

Reply:
yes